cluster/manifests/z-karpenter/webhooks.yaml

```diff
@@ -1,128 +0,0 @@
-{{ if eq .Cluster.ConfigItems.karpenter_pools_enabled "true"}}
```

Comment:
note: we should also make sure to delete these resources via `cluster/manifests/deletions.yaml` file. Currently they would not be deleted with `karpenter_pools_enabled=true`

Reply:
Same for the CRDs "renaming" there we should also delete the old ones via `deletions.yaml`

Reply:
fixed, 
the only thing that could not be done in this iteration is the CRDs.
they can not be deleted before removing their instances
so I kept them under the `karpenter_pools_enabled=true` for now until being cleaned up later after migration is complete